src/pages/workspaces/workspace/Dashboard/WorkspaceDescription.test.ts

```diff
@@ -152,11 +152,13 @@ describe('WorkspaceDescription', () => {
     const editButton = screen.getByLabelText('Edit description');
     await user.click(editButton);
 
-    onChange(newDescription);
-
+    act(() => {
```

Comment:
This one seems odd. I'm not clear why it is needed, but I also wonder why there is no `await`.